distributed/worker.py

```diff
@@ -2119,6 +2181,82 @@ def get_worker():
 
     See Also
     --------
+    get_client
+    worker_client
+    """
+    try:
+        return thread_state.execution_state['worker']
+    except AttributeError:
+        if _global_workers:
+            return _global_workers[-1]()
+        else:
+            raise
+
+
+def get_client(address=None):
+    """ Get a client while within a task
```

Comment:
Will it be asynchronous or not? It doesn't seem obvious that this always returns the same kind of client...